gravitee-apim-console-webui/src/management/api/runtime-logs-v4/runtime-logs-details/components/api-runtime-logs-connection-log-details/components/response-body-accordion/body-accordion.component.html

```diff
@@ -0,0 +1,26 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<cdk-accordion *ngIf="parsedBody">
+  <cdk-accordion-item class="accordion" #accordionItem="cdkAccordionItem" [attr.aria-expanded]="accordionItem.expanded">
+    <div class="accordion__header" (click)="accordionItem.toggle()">
+      <div class="accordion__title" (click)="accordionItem.toggle()">Body</div>
+      <mat-icon [svgIcon]="accordionItem.expanded ? 'gio:nav-arrow-down' : 'gio:nav-arrow-up'"></mat-icon>
+    </div>
+    <div *ngIf="!accordionItem.expanded" class="accordion__body">{{ parsedBody | json }}</div>
```

Comment:
what if the body is not json ? it is properly handled in the component but I think here we should avoid to use the pipe if not a json ?